src/dmd/globals.d

```diff
@@ -631,3 +633,19 @@ enum PINLINE : int
 alias StorageClass = uinteger_t;
 
 extern (C++) __gshared Global global;
+
+// Structure to hold details of an error
+public struct ErrorRecord
+{
+    Loc loc;
+    string type;
```

Comment:
type???

Reply:
one of `Error: `, `         ` (for supplemental), `Warning: `, or `Deprecation: `

Reply:
I should have said: "need documentation comment"

Reply:
Ahh, true.